src/analysis/complex/isometry.lean

```diff
@@ -49,6 +49,32 @@ def rotation : circle →* (ℂ ≃ₗᵢ[ℝ] ℂ) :=
 
 @[simp] lemma rotation_apply (a : circle) (z : ℂ) : rotation a z = a * z := rfl
 
+lemma reflection_rotation (a : circle) : rotation a ≠ conj_lie :=
+begin
+  intros h,
+  by_cases hu: (a:ℂ).re = -1,
+  { have : rotation a 1 = conj_lie 1,
+    { rw h, },
```

Comment:
The `linear_isometry_equiv.congr_fun` I describe below can replace this `have` and the one on line 61 too.